airgun/widgets.py

```diff
@@ -2685,6 +2686,34 @@ def read(self):
         return self.column_value.read()
 
 
+class FieldWithEditBtn(Widget):
+    """A pair of a field and a button that makes the field editable.
+    After editing, confirm by checkmark or cancel by X.
+    Is present e.g. in PF5 Settings.
+    """
+
+    ROOT = '//td[2]'
+    textinput = TextInput(locator=".//input[@data-ouia-component-type='PF5/TextInput']")
+    textarea = TextInput(locator=".//textarea")
+    dropdown = FormSelect(locator=".//select[@data-ouia-component-type='PF5/FormSelect']")
```

Comment:
same as above with the OUIA component, but with formSelect if compontent-id applies